styleguide/source/_patterns/01-molecules/09-search-results/search-results-best-match.twig

```diff
@@ -0,0 +1,19 @@
+{#
+ # @file search-results-best-match.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_one_up">
+  {% include 'atoms-h6' with { 'class' : 'search-result_label', 'content' : 'Best Match' } %}
+  <div class="search-result">
```

Comment:
I think this should be a block level a tag since the entire thing acts as a link